config/materials/ISO1629_SBR_SHORE_A70.mtl

```diff
@@ -0,0 +1,79 @@
+﻿ND_RelParSet_K01 = {
+
+Name = RUBBER
+
+PARAMETERS =
+{
+  Name = PTC_MATERIAL_DESCRIPTION
+  Type = String
+  Default = 'SBR rubbet. Shore 70.'
```

Comment:
The description contains a typo. For the shore I would specify that is SHORE A.
I would also add the ISO.

Reply:
Done!